torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -223,7 +226,15 @@ Value* CloneValueFromListConstruct(Value* v, std::shared_ptr<Graph> n_graph) {
       auto new_input = n_graph->addInput();
       new_input->copyMetadata(input);
       unsqueezed_node->addInput(new_input);
-      unsqueezed_node->is_(attr::axes, {0});
+      if (opset_version >= 13) {
```

Comment:
nit: Would it be better that we create a new method with a meaningful name to increase readability and let people know why we have these new code? Like, change this line to "if (unsqueezeNeedsMultipleParameters(opset_version))". The method is simple as:

bool unsqueezeNeedsMultipleParameters(int opset_version) {
    return opset_version >= 13;
}

We can put it into helper.h file so this method could be shared among different places, like shape_type_inference.cpp and peehole.cpp.